src/SeaIceDeformation/visualize_grid.py

```diff
@@ -0,0 +1,9 @@
+import utils_visualise_grid
+import utils_load_grid
+
+grid_dict = utils_load_grid.load_grid('/home/leki/ice-tracker-deformations/data/00_grid/cropped_grid.nc')
```

Comment:
Maybe it would be better to not hardcode this path here ? maybe using a relative path would be better...

Reply:
This was a temporary script I used early on, I've deleted it from my branch now. Thanks for catching it!